stdlib/websub/src/main/ballerina/src/websub/subscriber_client.bal

```diff
@@ -72,10 +77,10 @@ public type SubscriptionClient client object {
 
 # Function to build the subscription request to subscribe at the hub.
 #
-# + mode - Whether the request is for subscription or unsubscription
+# + mode - Whether the request is for registration or unregistration
```

Comment:
Conceptually this change is incorrect. Please use the terminology `subscription`